examples/Phonon_Caustics/g4cmpPhononCaustics.cc

```diff
@@ -0,0 +1,151 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+/// \file phonon/g4cmpPhonon.cc
+/// \brief Main program of the G4CMP/phonon example
+//
+// $Id$
+//
+// 20140509  Add conditional code for Geant4 10.0 vs. earlier
+// 20150112  Remove RM->Initialize() call to allow macro configuration
+// 20160111  Remove Geant4 version check since we now hard depend on 10.2+
+// 20170816  Add example-specific configuration manager
+// 20220718  Remove obsolete pre-processor macros G4VIS_USE and G4UI_USE
```

Comment:
Since this is a new and different file, please remove the past history that you copied from elsewhere.

Reply:
I removed all the history and now the program starts with the following lines:
/***********************************************************************\
 * This software is licensed under the terms of the GNU General Public *
 * License version 3 or later. See G4CMP/LICENSE for the full license. *
\***********************************************************************/

/// \file Phonon_Caustics/g4cmpPhononCaustics.cc
/// \brief Main program of the G4CMP/Phonon_Caustics example
//
// $Id$
//